### ISSUE TITLE
Field default value

### ISSUE THREAD

**Issue by laraujo7** [martinthenth/goal#119]
Hey there :wave: 

I have been using this library for some time and have loved it, but there is something that I wish I had: a default value for a field if one is not provided.
Something like:

`required :provider, :enum, values: providers(), default: :stripe`

**Comment by martinthenth** [comment]
Good idea! I merged your PR and will publish a new package. Thank you!
